build/yamls/antrea-aks.yml

```diff
@@ -174,6 +174,169 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
+metadata:
+  labels:
+    app: antrea
+  name: externalentities.core.antrea.tanzu.vmware.com
+spec:
+  group: core.antrea.tanzu.vmware.com
+  names:
+    kind: ExternalEntity
+    plural: externalentities
+    shortNames:
+    - ee
+    singular: externalentity
+  preserveUnknownFields: false
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            endpoints:
+              items:
+                properties:
+                  ip:
+                    format: ipv4
+                    type: string
+                  name:
+                    type: string
+                  ports:
+                    items:
+                      properties:
+                        name:
+                          type: string
+                        port:
+                          x-kubernetes-int-or-string: true
+                        protocol:
+                          type: string
+                      type: object
+                    type: array
+                type: object
+              type: array
+            externalNode:
+              type: string
+          type: object
+      type: object
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    app: antrea
+  name: networkpolicies.security.antrea.tanzu.vmware.com
+spec:
+  group: security.antrea.tanzu.vmware.com
+  names:
+    kind: NetworkPolicy
+    plural: networkpolicies
+    shortNames:
+    - anp
```

Comment:
the prefix "a" is by purpose? the full name doesn't have "Antrea".

Reply:
It is on purpose. AntreaNetworkPolicy is suggested by @jianjuns and @suwang48404. For the full name, I'll let @abhiraut comment on if we plan to simply keep it NetworkPolicy or change it to AntreaNetworkPolicy eventually.

Reply:
OK, I'm open to both names but want the full name and the short name consistent, i.e. AntreaNetworkPolicy with anp, or NetworkPolicy with np.
Are you ok with the mixed names? @jianjuns @suwang48404 @abhiraut

Reply:
I think NetworkPolicy as full name is good, but agreed anp as short name is a little confusing. However, K8s NP uses short name netpol, do you think ok to use "np"?

Reply:
Personally I prefer "netpol" to be consistent with K8s, and `antctl get` also uses netpol. I guess "np" is to map "cnp". I'm fine with having "netpol" for NetworkPolicy and "cnp" for ClusterNetworkPolicy.

Reply:
What will happen for kubectl get netpol?

Reply:
Same as `kubectl get networkpolicy`, it defaults to K8s networkpolicy, need to specify group suffix to get Antrea networkpolicies, e.g. `kubectl get netpol.security.antrea.tanzu.vmware.com` or `kubectl get netpol.secrurity`, the suffix doesn't need to be complete, just long enough to not be ambiguous.

Reply:
agree to keep NetworkPolicy as kind name.. so we could refer to them as Antrea NetworkPolicy .. 
for short name .. do `cnetpol` and `netpol` make sense? or `cnp` and `np` to not have to append groups suffix.. that way we are consistent across?

Reply:
I feel cnp is fine (cnetpol sounds a little strange to me).

Reply:
So is the decision to rename shortName to `netpol` here? @abhiraut @jianjuns

Reply:
lets do netpol and keep cnp as is .. i think both Quan and Jianjun also agree with this